src/canonmn_int.cpp

```diff
@@ -2460,26 +2505,28 @@ namespace Exiv2 {
     constexpr TagDetails canonRFLensType[] = {
         { 0, N_("n/a")   },
         { 257, N_("Canon RF 50mm F1.2L USM")  },
-        { 257, N_("Canon RF 24-105mm F4L IS USM")  },
-        { 257, N_("Canon RF 28-70mm F2L USM")  },
-        { 257, N_("Canon RF 35mm F1.8 MACRO IS STM")  },
-        { 257, N_("Canon RF 85mm F1.2L USM")  },
-        { 257, N_("Canon RF 85mm F1.2L USM DS")  },
-        { 257, N_("Canon RF 24-70mm F2.8L IS USM")  },
-        { 257, N_("Canon RF 15-35mm F2.8L IS USM")  },
-        { 257, N_("Canon RF 24-240mm F4-6.3 IS USM")  },
-        { 257, N_("Canon RF 70-200mm F2.8L IS USM")  },
-        { 257, N_("Canon RF 85mm F2 MACRO IS STM")  },
-        { 257, N_("Canon RF 600mm F11 IS STM")  },
-        { 257, N_("Canon RF 600mm F11 IS STM + RF1.4x")  },
-        { 257, N_("Canon RF 600mm F11 IS STM + RF2x")  },
-        { 257, N_("Canon RF 800mm F11 IS STM")  },
-        { 257, N_("Canon RF 800mm F11 IS STM + RF1.4x")  },
-        { 257, N_("Canon RF 800mm F11 IS STM + RF2x")  },
-        { 257, N_("Canon RF 24-105mm F4-7.1 IS STM")  },
-        { 257, N_("Canon RF 100-500mm F4.5-7.1L IS USM")  },
-        { 257, N_("Canon RF 100-500mm F4.5-7.1L IS USM + RF1.4x")  },
-        { 257, N_("Canon RF 100-500mm F4.5-7.1L IS USM + RF2x")  }
+        { 258, N_("Canon RF 24-105mm F4L IS USM")  },
+        { 259, N_("Canon RF 28-70mm F2L USM")  },
+        { 260, N_("Canon RF 35mm F1.8 MACRO IS STM")  },
+        { 261, N_("Canon RF 85mm F1.2L USM")  },
+        { 262, N_("Canon RF 85mm F1.2L USM DS")  },
+        { 263, N_("Canon RF 24-70mm F2.8L IS USM")  },
+        { 264, N_("Canon RF 15-35mm F2.8L IS USM")  },
+        { 265, N_("Canon RF 24-240mm F4-6.3 IS USM")  },
+        { 266, N_("Canon RF 70-200mm F2.8L IS USM")  },
+        { 267, N_("Canon RF 85mm F2 MACRO IS STM")  },
+        { 268, N_("Canon RF 600mm F11 IS STM")  },
+        { 269, N_("Canon RF 600mm F11 IS STM + RF1.4x")  },
+        { 270, N_("Canon RF 600mm F11 IS STM + RF2x")  },
+        { 271, N_("Canon RF 800mm F11 IS STM")  },
+        { 272, N_("Canon RF 800mm F11 IS STM + RF1.4x")  },
+        { 273, N_("Canon RF 800mm F11 IS STM + RF2x")  },
+        { 274, N_("Canon RF 24-105mm F4-7.1 IS STM")  },
+        { 275, N_("Canon RF 100-500mm F4.5-7.1L IS USM")  },
+        { 276, N_("Canon RF 100-500mm F4.5-7.1L IS USM + RF1.4x")  },
+        { 277, N_("Canon RF 100-500mm F4.5-7.1L IS USM + RF2x")  },
+        { 278, N_("Canon RF 70-200mm F4L IS USM")  },
+        { 280, N_("Canon RF 50mm F1.8 STM")  }
```

Comment:
this closes #2059, just mentioning it so we don't forget

Reply:
@hassec I believe this is included in the PR.  I've linked #2059 to this PR so it closed when this is merged.

```bash
581 rmills@rmillsm1:~/gnu/github/exiv2/main/build $ grep -n 'Canon RF 50mm F1.8 STM' ../src/*.cpp 
../src/canonmn_int.cpp:2019:        {61182, "Canon RF 50mm F1.8 STM"                                    }, // 8
../src/canonmn_int.cpp:2477:        { 280, N_("Canon RF 50mm F1.8 STM")  }
582 rmills@rmillsm1:~/gnu/github/exiv2/main/build $ 
```

Do  you know why it's in two places, @postscript-dev ?  It's localised in one `N_("....")` and not in the other.